### DISCUSSION TITLE
N64 Blueretro Doesn't connect on Bluetooth for OTA. Controller pak not detected in games.

### DISCUSSION THREAD

**Discussion by Emagid00** [darthcloud/BlueRetro#543] [Q&A]
The controller is unable to connect for updates here: https://blueretro.io/index.html
Somehow yesterday I was able to get it to detect the adapter, but it would continue to get a network error message. As of today, it no longer detects the adapter at all.
Also, the controller pak is not detected in any games that use it. Is my adapter defective?

**Comment by Emagid00** [comment]
Update: After plugging the adapter into port 2, it is now detected again. But I keep getting the following error message when trying to connect: 

Connecting to GATT Server...
NetworkError
Argh! Couldn't connect to BlueRetro

**Comment by darthcloud** [comment]
You got a N64 digital?
If so you need to install an HW fix in the console see details in this thread #264 .

Press your controller Home/PS/Xbox buttons to toggle between controller pak / rumble pak

For web connection,
Make sure you don't have Bluetooth LED strip/ smart light that might connect to BlueRetro.

Then try moving closer to the dongle, use different computer or phone. Close/reopen browser/tab, try multiple time.

**Reply by Emagid00** [reply]
No, I don't have an N64 Digital install. It does have an Advanced RGB kit installed however. 

I tried using a different controller and was able to switch the memory pak/rumble successfully. For some reason it doesn't seem to work when using the NSO N64 controller. It just stays in rumble mode no matter how many times I press the home button.

Sorry I wasn't clear, but this is the external Blueretro N64 adapter, not internal. I made sure to disconnect all controllers and the adapter just won't connect. Consol is right next to the computer so don't think distance could be an issue. Unfortunately I don't have an Android phone to try it with. Doesn't seem to work on my IOS device.

**Reply by Emagid00** [reply]
OK, I think I missed that you need to hit Home each time you restart the system to go back and forth between Mem/Rumble Pak. Seems to be working OK with the NSO controller now.

Unfortunately no matter how many times I try to update the FW however, it just won't connect.

**Reply by darthcloud** [reply]
If that Advanced RGB kit got an OSD menu the same bug and fix as N64 Digital likely apply.

For the web latest update got some improvement, but to get it install you will likely need to find/borrow another laptop.

**Reply by Emagid00** [reply]
Thanks. It worked fine with my work laptop. I'm surprised it didn't work on my main PC as it's pretty recent too, but glad I got it working at least.
